submissions/ArthurGorbenko/html.css.practice/index.html

```diff
@@ -0,0 +1,154 @@
+                      <li class="card">
+                          <a href="#" class="icon videoeditor">
+                                <span class="title_btn">VideoEditor</span>
+                           </a>
+                       </li> 
+                      <li class="card">
+                          <a href="#" class="icon tools">
+                           <span class="title_btn">Tools</span>
+                           </a>
+                       </li> 
+                       <li class="card">
+                          <a href="#" class="icon translate">
+                           <span class="title_btn">Translate</span>
+                           </a>
+                       </li> 
+                       <li class="card">
+                          <a href="#" class="icon mail">
+                           <span class="title_btn">Mail</span>
+                           </a>
+                       </li> 
+                  </ul>
+
+                <div class="even_more"><a href="" class="link">Even more from G</a></div>
+            </div>
+        <label for="more" class="button more_label">
+            <div class="more_button">More</div>
+        </label>
+      <ul class="list_1">
+       <li class="card">
+           <a href="#" class="icon board">
+                <span class="title_btn">Board</span>
+            </a>
+        </li>
+       <li class="card">
+           <a href="#" class="icon bookmarks">
+                <span class="title_btn">Bookmarks</span>
+            </a>
+        </li>
+       <li class="card">
+           <a href="#" class="icon calendar">
+                    <span class="title_btn">Calendar</span>
+            </a>
+        </li>
+       <li class="card">
+           <a href="#" class="icon diagram">
+            <span class="title_btn">Diagram</span> 
+          </a>
+        </li> 
+       <li class="card">
+           <a href="#" class="icon implement">
+            <span class="title_btn">Implement</span> 
+           </a>
+        </li> 
+       <li class="card">
+           <a href="#" class="icon icq">
+            <span class="title_btn">Messenger</span> 
+           </a>
+        </li> 
+       <li class="card">
+           <a href="#" class="icon security">
+                <span class="title_btn">Security</span>
+            </a>
+        </li> 
+       <li class="card">
+           <a href="#" class="icon tables">
+                <span class="title_btn">Tables</span>
+             </a>
+        </li> 
+       <li class="card">
+           <a href="#" class="icon maps">
+                    <span class="title_btn">Maps</span>
+            </a>
+        </li> 
+       <li class="card">
+           <a href="#" class="icon documents">
+            <span class="title_btn">Document</span>
+            </a>
+        </li> 
+    </ul>
+   </div>
+        <div class="ring"><a href="#" class="link"><img src="notification.png" class="icon ring_img"></a></div>
```

Comment:
Add and fill `alt` attribute in the `<img>`tag.

Reply:
Done.